src/_pytest/fixtures.py

```diff
@@ -256,15 +275,16 @@ def get_parametrized_fixture_keys(item: nodes.Item, scope: Scope) -> Iterator[_K
         for argname, param_index in sorted(cs.indices.items()):
             if cs._arg2scope[argname] != scope:
                 continue
+            param = SafeHashWrapper(cs.params.get(argname, param_index))
```

Comment:
Why is the `get` needed here?

Reply:
Wasn't sure if `argname` is guaranteed to be in `cs.params`. But now I think it is (and if not, it would be y pytest bug), so get is not required.